src/aks-preview/azext_aks_preview/commands.py

```diff
@@ -179,6 +186,14 @@ def load_command_table(self, _):
         g.custom_command('delete', 'aks_snapshot_delete',
                          supports_no_wait=True)
 
-    # AKS trusted access roles commands
+    # AKS trusted access role commands
     with self.command_group('aks trustedaccess role', trustedaccess_role_sdk, client_factory=cf_trustedaccess_role) as g:
         g.custom_command('list', 'aks_trustedaccess_role_list')
+
+    # AKS trusted access rolebinding commands
+    with self.command_group('aks trustedaccess rolebinding', trustedaccess_role_binding_sdk, client_factory=cf_trustedaccess_role_binding) as g:
+        g.custom_command('list', 'aks_trustedaccess_role_binding_list')
+        g.custom_command('get', 'aks_trustedaccess_role_binding_get')
+        g.custom_command('create', 'aks_trustedaccess_role_binding_create_or_update', supports_no_wait=True)
+        g.custom_command('update', 'aks_trustedaccess_role_binding_create_or_update', supports_no_wait=True)
+        g.custom_command('delete', 'aks_trustedaccess_role_binding_delete', supports_no_wait=True)
```

Comment:
Please declare wait command and add help info for it since create/update/delete support `--no-wait`.

Reply:
No need to declare wait command for it is declared under parent group: `self.command_group('aks'`  ?

Reply:
Removed the supports_no_wait, for the commands are fast and no need this parameter